src/main/java/com/ssafy/ssafybox/crud/controller/CRUDController.java

```diff
@@ -0,0 +1,56 @@
+package com.ssafy.ssafybox.crud.controller;
+
+import com.ssafy.ssafybox.crud.model.dto.RequestDto;
+import com.ssafy.ssafybox.crud.model.dto.ResponseDto;
+import com.ssafy.ssafybox.crud.model.dto.TodoDto;
+import com.ssafy.ssafybox.crud.model.service.CRUDService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/todos")
+public class CRUDController {
+
+    private final CRUDService CRUDService;
+
+    @Autowired
+    public CRUDController(CRUDService CRUDService) {
+        this.CRUDService = CRUDService;
+    }
+
+    // GET 요청: 할 일 조회
+    @GetMapping()
```

Comment:
```suggestion
    @GetMapping
```
매핑에 아무것도 없을 때 괄호 생략해도 돼요!